### ISSUE TITLE
Determine which fields are needed for 'batches' and 'batch_items' tables.

### ISSUE THREAD

**Issue by afred** [samvera-labs/hyrax-batch_ingest#3]
**Background:** Batch ingests are tracked in a relational database with two tables: `batches`, and `batch_items`, where a `Batch` has many `BatchItems`.

We need to figure out what the fields for those tables are. 

Things to consider:
  * A `BatchItem` may represent more than one record in the persistence layer and/or more than one derivative binary file. So having a field for tracking the results of each batch item is necessary.

See Donut as a reference: https://github.com/nulib/donut/blob/master/db/schema.rb#L18-L36

**Done when:**
  1. Fields for `batch` and `batch_item` tables has been determined.
  1. New ticket is created for actually creating tables in the db migration code.

depends on #12.

**Comment by cjcolvar** [comment]
Depends on #16.
